<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_dsnu_width">
      <property name="text">
       <string>Width</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_dsnu_width"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="label_dsnu_height">
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_dsnu_height"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <widget class="QLabel" name="label_dsnu_sensorbit">
      <property name="text">
       <string>SensorBit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_dsnu_sensorbit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>95</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_18">
    <item>
     <widget class="QLabel" name="label_dsnu_cfa">
      <property name="text">
       <string>CFA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_dsnu_cfa"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>123</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_19">
    <item>
     <widget class="QLabel" name="label_dsnu_widthpercent">
      <property name="text">
       <string>WidthPercent</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_dsnu_widthpercent"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>129</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_20">
    <item>
     <widget class="QLabel" name="label_dsnu_heightpercent">
      <property name="text">
       <string>HeightPercent</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_dsnu_heightpercent"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_21">
    <item>
     <widget class="QLabel" name="label_dsnu_time_image1">
      <property name="text">
       <string>Time_image1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_dsnu_time_image1"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_12">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_22">
    <item>
     <widget class="QLabel" name="label_dsnu_time_image2">
      <property name="text">
       <string>Time_image2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_dsnu_time_image2"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_dsnu_calculate">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_13">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_23">
    <item>
     <widget class="QLabel" name="label_dsnu_value">
      <property name="text">
       <string>DSNU(/sec)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_dsnu_value"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_dsnu_image1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>image1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dsnu_image2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>image2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
